gtk/gtkcssenumvalueprivate.h

```diff
@@ -34,7 +34,8 @@ GtkBorderStyle  _gtk_css_border_style_value_get       (const GtkCssValue *value)
 GtkCssValue *   _gtk_css_font_size_value_new          (GtkCssFontSize     size);
 GtkCssValue *   _gtk_css_font_size_value_try_parse    (GtkCssParser      *parser);
 GtkCssFontSize  _gtk_css_font_size_value_get          (const GtkCssValue *value);
-double          _gtk_css_font_size_get_default        (GtkStyleProviderPrivate *provider);
+double          gtk_css_font_size_get_default_px      (GtkStyleProviderPrivate *provider,
```

Comment:
@ptomato nope, but private functions are not being marked with underscores, just kept in private header files.

I guess so you don't have to do a massive find a replace every time you make private api public? I think the underscore was used in some old makefile rule that's not needed anymore.